packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -490,11 +490,40 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   }
 
   void _handleRightClickDown(TapDownDetails details) {
+    final Offset? previousSecondaryTapDownPosition = lastSecondaryTapDownPosition;
+    final bool toolbarIsVisible = _selectionOverlay?.toolbarIsVisible ?? false;
     lastSecondaryTapDownPosition = details.globalPosition;
     widget.focusNode.requestFocus();
-    _selectWordAt(offset: details.globalPosition);
-    _showHandles();
-    _showToolbar(location: details.globalPosition);
+    switch (defaultTargetPlatform) {
```

Comment:
Is there logic here that should be deduplicated with EditableText (maybe [here](https://github.com/flutter/flutter/blob/ceeaf98e64703a50cf3cfcf0bd103de98b979a45/packages/flutter/lib/src/widgets/text_selection.dart#L2419))?  Or maybe just in the long run.

Reply:
We talked offline.  Deduplication with EditableText will be a long term thing.